packages/calling-stateful-client/src/CallContext.ts

```diff
@@ -570,7 +570,7 @@ export class CallContext {
   public setDeviceManagerCameras(cameras: VideoDeviceInfo[]): void {
     this.setState(
       produce(this._state, (draft: CallClientState) => {
-        if ((!draft.deviceManager.cameras || draft.deviceManager.cameras.length === 0) && cameras.length > 0) {
+        if (cameras.length > 0 && !cameras.some((camera) => camera.id === draft.deviceManager.selectedCamera?.id)) {
```

Comment:
Please add comment explaining why this is done in the code. Essentially, copy the PR description into the code comment.

The SDK behaviour is highly unusual, and will definitely trip me up the next time I read this :/

Reply:
Have we asked the SDK team why we get a dummy camera in the first place? That doesn't sound right.

Reply:
Sure added a comment and I just let them know about this potential bug here https://teams.microsoft.com/l/message/19:ea8322f638f34acdb74162cacd55c92b@thread.skype/1624308079847?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e9c1fc3-39df-4486-a26a-456d80e80f82&parentMessageId=1624308079847&teamName=Azure%20Communication%20Services&channelName=Calling&createdTime=1624308079847